docs/ref/django-admin.txt

```diff
@@ -722,6 +722,16 @@ be warned that using ``--fake`` runs the risk of putting the migration state
 table into a state where manual recovery will be needed to make migrations
 run correctly.
 
+.. versionadded:: 1.8
+
+.. django-admin-option:: --fake-initial
+
+The ``--fake-initial`` option can be used to allow Django to skip an app's
```

Comment:
Maybe say something here about the intended purpose of the option? "This option is intended for use when first running migrations against a database that pre-existed the use of migrations."